src/components/SystemsTable/ConnectionStatusCol.js

```diff
@@ -172,7 +172,7 @@ const ConnectionStatusColumn = ({ connection_status, executor_type }) => {
               </p>{' '}
               permission within Remote Host Configuration Manager.
               <a
-                href="https://access.redhat.com/documentation/en-us/red_hat_insights/1-latest/html/red_hat_insights_remediations_guide/host-communication-with-insights_red-hat-insights-remediation-guide"
+                href="https://console.redhat.com/insights/connector "
```

Comment:
<img width="594" alt="Screenshot 2024-06-11 at 3 23 05 PM" src="https://github.com/RedHatInsights/insights-remediations-frontend/assets/87209745/fe82dcdb-b4d1-426e-9b65-c697e12e2fa5">

One more minor thing.. Should we remove the first non-link `Remote Host Configuration Manager.` ? So it's not repeated?

Reply:
Just sent it up